File: P04 - Inherirance/Java/TestMobildanKereta.java
package Praktikum.P04.P04c_Latihan_Inheritance_Java_227008_Elmosius_Suli;
// File : TestMobildanKereta.java
// Nama : Elmosius Suli
// NRP  : 2272008
// Kelas : B
// Ket :
public class TestMobildanKereta {
    public static void main(String[] args) {
        Mobil sedan = new Mobil();

        System.out.println("Sedanku:");
        System.out.println("Kecepatan Maks: " + sedan.kecepatanMaks);
        System.out.println("Kapasitas Penumpang: " + sedan.kapasitasPenumpang);
        System.out.println("Bahan Bakar: " + sedan.bahanBakar);
        System.out.println("Jenis Mesin: " + sedan.mesin);

        Kereta andong = new Kereta();
        System.out.println("\nAndongku:");
        System.out.println("Kecepatan Maks: " + andong.kecepatanMaks);
        System.out.println("Kapasitas Penumpang: " + andong.kapasitasPenumpang);
        System.out.println("Tenaga Penarik: " + andong.tenagaPenarik);
        System.out.println("Jumlah Penarik: " + andong.jmlPenarik);


    }
}

File: P04 - Inherirance/Python/TestMobildanKereta.py
# File TestMobildanKereta.py

from Kendaraan import Kendaraan
from Mobil import Mobil
from Kereta import Kereta

def main():
    sedan = Mobil()
    print("Sedanku:")
    print("Kecepatan Maks: " + str(sedan._kecepatan_maks)) 
    print("Kapasitas Penumpang: " + str(sedan._kapasistas_penumpang))
    print("Bahan Bakar: " + sedan.bahan_bakar)
    print("Jenis Mesin: " + sedan.mesin)
    print("Jumlah kendaraan: " + str(Kendaraan.jml_kendaraan))
    
    andong = Kereta()
    print("\nAndongku:")
    print("Kecepatan Maks: " + str(andong._kecepatan_maks)) 
    print("Kapasitas Penumpang: " + str(andong._kapasistas_penumpang))
    print("Tenaga Penarik: " + andong.tenaga_penarik)
    print("Jumlah Penarik: " + str(andong.jml_penarik))
    print("Jumlah kendaraan: " + str(Kendaraan.jml_kendaraan))
    
if __name__ == "__main__":
    main()